test_util/main.tf

```diff
@@ -33,6 +33,17 @@ variable "windowsagent_num" {
   description = "Defines the number of Windows agents for the cluster."
 }
 
+variable "ssh_public_key_file" {
+  type = "string"
+  default = "./tf-dcos-rsa.pem.pub"
```

Comment:
This is not a good default IMHO. It should be `~/.ssh/id_rsa.pub`. Remember that we want to use the `main.tf` locally as well.

Reply:
That's why I allow configuration. I'd not like to break CI

Reply:
Ok. I'll change it in the Makefile then.